hw/ip/aon_timer/doc/interfaces.md

```diff
@@ -23,10 +23,10 @@ Referring to the [Comportable guideline for peripheral device functionality](htt
 
 ## Interrupts
```

Comment:
I'd suggest squashing this commit ("Autogenerated documentation") with the previous one. I think the result ends up without an inconsistent state in the history?

Reply:
Thanks, done now.